### ISSUE TITLE
[5.0.0-RC-2][BUG] Service-API errors when user does not have a photo

### ISSUE THREAD

**Issue by ivan-sukhomlyn** [reportportal/reportportal#773]
`Service-API` produce errors when the user does not have a photo.
I think that it's not an error and the user can be without an avatar or have some random avatar on another hand.

**Service-API logs:**
```java
2019-12-04 15:29:46.184  WARN 1 --- [nio-8585-exec-4] c.e.t.r.c.e.rest.RestExceptionHandler    : Resolved [com.epam.ta.reportportal.exception.ReportPortalException: Error in handled Request. Please, check specified parameters: 'User - 'some_user' does not have a photo.']
2019-12-04 15:30:20.956 ERROR 1 --- [nio-8585-exec-6] c.e.t.r.c.e.rest.RestExceptionHandler    : Handled error: 

com.epam.ta.reportportal.exception.ReportPortalException: Error in handled Request. Please, check specified parameters: 'User - 'some_user' does not have a photo.'
	at com.epam.ta.reportportal.binary.impl.UserBinaryDataServiceImpl.lambda$loadUserPhoto$1(UserBinaryDataServiceImpl.java:98)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at com.epam.ta.reportportal.binary.impl.UserBinaryDataServiceImpl.loadUserPhoto(UserBinaryDataServiceImpl.java:97)
	at com.epam.ta.reportportal.core.file.impl.GetFileHandlerImpl.getUserPhoto(GetFileHandlerImpl.java:74)
	at com.epam.ta.reportportal.ws.controller.FileStorageController.getUserPhoto(FileStorageController.java:90)
	at com.epam.ta.reportportal.ws.controller.FileStorageController$$FastClassBySpringCGLIB$$ba8a78c4.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:749)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:295)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.security.access.intercept.aopalliance.MethodSecurityInterceptor.invoke(MethodSecurityInterceptor.java:69)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:688)
	at com.epam.ta.reportportal.ws.controller.FileStorageController$$EnhancerBySpringCGLIB$$ca0d5540.getUserPhoto(<generated>)
	at sun.reflect.GeneratedMethodAccessor498.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:190)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:138)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:104)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:892)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:797)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1039)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:942)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1005)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:897)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:634)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:882)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
```

**Comment by svettsova** [comment]
@ivan-sukhomlyn could you specify a version of RP and specific steps?

**Comment by evjlobanova** [comment]
Will be fixed in version 5.3

**Comment by ivan-sukhomlyn** [comment]
@svettsova I've provided ReportPortal's service name and version in the title.
service - **API**
service version - **5.0.0-RC-2**

**Comment by e-olga** [comment]
The issue is fixed and will be available with RP v5.3 released.
